source/js/components/expander.js

```diff
@@ -20,22 +20,38 @@ var Panel = React.createClass({
     }
     this.props.activateKey(itemKey);
   },
-  render: function() {
+  propTypes: {
```

Comment:
Could we move `propTypes:` up so it's the first property of `<Pane>`? Also I think we should make `itemKey` and `activeKey` required `props` as well.